src/biogeophys/PhotosynthesisMod.F90

```diff
@@ -3162,9 +3173,8 @@ subroutine PhotosynthesisHydraulicStress ( bounds, fn, filterp, &
                kp25_sha = kp25top * nscaler_sha
 
                ! Adjust for temperature
-
-               vcmaxse = 668.39_r8 - 1.07_r8 * min(max((t10(p)-tfrz),11._r8),35._r8)
-               jmaxse  = 659.70_r8 - 0.75_r8 * min(max((t10(p)-tfrz),11._r8),35._r8)
+               vcmaxse = 486.0_r8
+               jmaxse  = 495.0_r8
```

Comment:
- [ ] Should  not be changed  for Kattge.

Reply:
Yes, Kattge uses acclimation (lines in red), but Leuning does not so the values are just constants.